### ISSUE TITLE
This error occurs when I am trying vento.

### ISSUE THREAD

**Issue by sivierostein** [TiagoSilvaPereira/vemto-issues#315]
I am trying a simple app.  2 models  Sala (Classroom) and Aluno (student) . Sala has many Aluno and Aluno belongs to Sala relations.
When I try to add a hasmany field in the CRUD of Sala.  The error occur.

```
TypeError: Cannot read property 'type' of undefined
    at Function.getCorrectInputFromFieldAndCrud (app://./js/chunk-common.d5e10964.js:1:64716)
    at Proxy.addInputToCrud (app://./js/chunk-common.d5e10964.js:1:24879)
    at app://./js/chunk-common.d5e10964.js:1:33662
    at Array.forEach (<anonymous>)
    at Proxy.addInputsFromModel (app://./js/chunk-common.d5e10964.js:1:33649)
    at Function.fromModel (app://./js/chunk-common.d5e10964.js:1:32986)
    at Function.createFromCrudAndRelationship (app://./js/chunk-common.d5e10964.js:1:49444)
    at o.create (app://./js/project_crud.a0c809af.js:1:60832)
    at app://./js/project_crud.a0c809af.js:1:60793
```

---

vemto 1.1.4
Electron 8.5.5
win32 10.0.19044
Locale: pt-BR

**Comment by TiagoSilvaPereira** [comment]
Hi @sivierostein please check #283 for a workaround

**Comment by TiagoSilvaPereira** [comment]
Fixed on 1.1.5
